jwt-ballerina/src/jwt/inbound_jwt_auth_provider.bal

```diff
@@ -116,6 +118,14 @@ function setInvocationContext(string credential, JwtPayload jwtPayload) {
             if (scopeString is string && scopeString != "") {
                 auth:setInvocationContext(scopes = stringutils:split(scopeString, " "));
             }
+        } else if (claims.hasKey("scp")) {
```

Comment:
If you do an early return in #L114, Then code width can be reduced. Just a suggestion
```
        map<json>? customClaims = jwtPayload?.customClaims;
        if customClaims is () {
            return;
        }

        map<json> claims = <map<json>>customClaims;
        auth:setInvocationContext(claims = claims);
        if (claims.hasKey("scope")) {
            json scopeString = claims["scope"];
            if (scopeString is string && scopeString != "") {
                auth:setInvocationContext(scopes = stringutils:split(scopeString, " "));
            }
            return;
        }
        
        if (claims.hasKey("scp")) {
            json scopeString = claims["scp"];
            if (scopeString is json[] && scopeString.length() > 0) {
                string[]|error scopes = scopeString.cloneWithType(StringArray);
                if (scopes is string[]) {
                    auth:setInvocationContext(scopes = scopes);
                }
            }
        }
```

Reply:
Thanks for the suggestion. Fixed with https://github.com/ballerina-platform/module-ballerina-jwt/pull/27/commits/ef9669f9cfe271f2d568109dd30d5c837f02a0e6